.idea/zulip.iml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.dart_tool" />
+      <excludeFolder url="file://$MODULE_DIR$/.pub" />
+      <excludeFolder url="file://$MODULE_DIR$/build" />
+      <excludeFolder url="file://$MODULE_DIR$/ios/.symlinks" />
+      <excludeFolder url="file://$MODULE_DIR$/linux/flutter/ephemeral" />
+      <excludeFolder url="file://$MODULE_DIR$/macos/Flutter/ephemeral" />
+      <excludeFolder url="file://$MODULE_DIR$/packages/zulip_plugin/.dart_tool" />
+      <excludeFolder url="file://$MODULE_DIR$/packages/zulip_plugin/.pub" />
+      <excludeFolder url="file://$MODULE_DIR$/packages/zulip_plugin/build" />
+      <excludeFolder url="file://$MODULE_DIR$/scripts/icons/node_modules" />
```

Comment:
Ha, what this should actually be (contra https://github.com/zulip/zulip-flutter/pull/637#discussion_r1583753412 ) is:

```suggestion
      <excludeFolder url="file://$MODULE_DIR$/tools/icons/node_modules" />
```

There is no `scripts/`; that must have been… aha, it's what `tools/` was called before e5d1e3bf6f5b8c8cfc931282ac8f15ab4289670e.

If we'd had this file in those versions of the tree, we'd want it to say `scripts/`; but we didn't, and at this point it should say `tools/`.

---

My local `zulip.iml` has both; it was just hard to spot with all the noise from `.dart_tool`, `.pub`, and `build` in every plugin package. To make sure there's nothing else hiding there, here it is with all the noise stripped out:
```
$ grep -vFe '/.dart_tool"' -e '/.pub"' -e '/build"' zulip.iml 
<?xml version="1.0" encoding="UTF-8"?>
<module type="JAVA_MODULE" version="4">
  <component name="NewModuleRootManager" inherit-compiler-output="true">
    <exclude-output />
    <content url="file://$MODULE_DIR$">
      <excludeFolder url="file://$MODULE_DIR$/scripts/icons/node_modules" />
      <excludeFolder url="file://$MODULE_DIR$/linux/flutter/ephemeral" />
      <excludeFolder url="file://$MODULE_DIR$/tools/icons/node_modules" />
      <excludeFolder url="file://$MODULE_DIR$/windows/flutter/ephemeral" />
    </content>
    <orderEntry type="sourceFolder" forTests="false" />
    <orderEntry type="library" name="Dart SDK" level="project" />
    <orderEntry type="library" name="Flutter Plugins" level="project" />
    <orderEntry type="library" name="Dart Packages" level="project" />
  </component>
</module>
```

So, looks like everything else there is covered in this revision. There is this line:
```xml
    <orderEntry type="library" name="Dart Packages" level="project" />
```
But I can't find anything missing as a consequence of leaving that line out. For example…

… hmm no. I tried a search and found it did get results from "Dart Packages":
![image](https://github.com/zulip/zulip-flutter/assets/28173/7a23dc63-16e8-4f94-b613-44faf02fe03a)

But then I found that that entry had been added back in somehow:
```diff
--- .idea/zulip.iml
+++ .idea/zulip.iml
@@ -18,5 +18,6 @@
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" name="Dart SDK" level="project" />
     <orderEntry type="library" name="Flutter Plugins" level="project" />
+    <orderEntry type="library" name="Dart Packages" level="project" />
   </component>
 </module>
\ No newline at end of file
```

I discarded that with `git reset --hard`, tried again, and now I'm only seeing the results that can also be found under "Flutter Plugins":
![image](https://github.com/zulip/zulip-flutter/assets/28173/a97b90f3-ac68-4e7c-bd2e-31d67ded311e)

Similarly try searching for "Table" (a class from `package:drift`); it doesn't appear.

Or in fact something from the Flutter framework, like "Widget".

So yeah, that's a line we want to include. :slightly_smiling_face: